src/reducers-actions/combineReducers.ts

```diff
@@ -0,0 +1,77 @@
+import { Reducer } from 'react'
```

Comment:
Got a bit lost in this file. Is this something copied from a library or something? Is there a dependency we can use?